YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -294,11 +299,16 @@ Repository::Repository(const std::string& path)
 
     if (repo_already_exists)
     {
+        include_idb_ = is_file_tracked(get_original_idb_path());
```

Comment:
should it be get_original_idb_name instead ?

Reply:
I think that git is fine with either relative or absolute paths, so this shouldn't matter.

Reply:
It doesn't work on windows, I'll look at it

Reply:
_merci beaucoup_!